torch/csrc/jit/tensorexpr/external_functions.cpp

```diff
@@ -1368,7 +1368,7 @@ void nnc_aten_addmm(
   int64_t alpha = extra_args[0], beta = extra_args[1];
```

Comment:
```suggestion
  int64_t beta = extra_args[0], alpha = extra_args[1];
```

In our test (test_external_calls.cpp), it looks like it reads the first arg as beta anyway, so the math actually works out the original way + the variables are just poorly named.

Reply:
OK, rename them.